### ISSUE TITLE
[feat] files recommended configuration

### ISSUE THREAD

**Issue by nicoxygen** [OpenZWave/Zwave2Mqtt#234]
hello is it possible to create a file with inside a recommended configuration for each zwave module in order to have a turnkey configuration and put a link recommended configuration in the configuration tab

**Comment by robertsLando** [comment]
@nicoxygen Could you better explain what you mean please?

**Comment by nicoxygen** [comment]
so I'm talking about the advanced configuration (bit) of the z-wave modules
it would be good to create a file with each configuration set differently from the default configuration of the z-wave module and put a button (recommended configuration) above the configuration which would replace the default configuration. Jeedom already has that.
In the configuration file, you should be able to put several configurations of the same module, for example the philio allows you to set the temperatures in ° C or ° F
recommended configuration: Example my philio psm02-1 module works with these parameters:
bit5: 8
bit6: 4
bit7: 6
bit12: 3
bit13: 2
so I have all the information that goes back correctly

**Comment by robertsLando** [comment]
@nicoxygen THis means to create a device db with all recommended configurations, do you know how many zwave devices exists?

**Comment by nicoxygen** [comment]
Yes there are a lot but we can do this as the device.js if a recommended configuration exists for the device it will update otherwise nothing would happen is it possible? i have join pictures
![advanced-configuration](https://user-images.githubusercontent.com/17491075/74021382-b85bbd80-499b-11ea-8de9-9ed549f93623.png)

**Comment by robertsLando** [comment]
I make it in my todos but I can't ensure you when and if it will be implemented
